pinot-hadoop/pom.xml

```diff
@@ -72,8 +72,18 @@
           <groupId>commons-lang</groupId>
           <artifactId>commons-lang</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>com.linkedin.pinot</groupId>
+          <artifactId>pinot-common</artifactId>
+        </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>com.linkedin.pinot</groupId>
+      <artifactId>pinot-common</artifactId>
+      <version>0.016</version>
```

Comment:
I don't think we should hard code the version.

Reply:
good call, removed